docs/installation-guide/quickstart.rst

```diff
@@ -132,28 +147,29 @@ as a parameter with each kubectl command.
    kube-system   registry-bootstrap                            1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
    kube-system   salt-master-bootstrap                         2/2       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
 
-Adding a master to the cluster
-------------------------------
+Add a master to the cluster
+---------------------------
 
 Now it's time to add more nodes to the cluster. First you need to add
-2 nodes with etcd and master roles to improv redundancy of
+2 nodes with etcd and master roles to improve redundancy of
 the control-plane. Here is the procedure to add one, simply do it
 twice to have 3 masters (bootstrap + 2 new master).
 
-1. Copy the ssh-key to the new master node
+1. Copy the ssh-key to the new control-plane node
 
 .. code-block:: shell
 
-   $ ssh-copy-id <IP-of-the-new-master-node>
+   $ ssh-copy-id <IP-of-the-new-control-plane-node>
 
-2. Create a YAML config file :file:`new-master-node.yaml` for this new master.
+2. Create a :term:`node manifest <Kubernetes node manifest>` for
+   this new control-plane node as :file:`new-control-plane-node.yaml`.
```

Comment:
I suggest we remind this file has to be created on the bootstrap node

Reply:
Actually, this part doesn't need to happen on the bootstrap node ; I'll try to make it clear.